doc/user/content/manage/dbt/_index.md

```diff
@@ -358,16 +357,28 @@ SELECT
 FROM {{ ref('kafka_topic_a') }}
 ```
 
-The model above would be compiled to `database.schema.view_a`. One thing to note
-here is that the model depends on the Kafka source defined above. To express
-this dependency and track the **lineage** of your project, you can use the dbt
-[`ref()`](https://docs.getdbt.com/reference/dbt-jinja-functions/ref) function.
+The model above would be compiled to `database.schema.view_a`.
+
+{{< tip >}}
+
+- The model depends on the Kafka source defined above. To express this
+  dependency and track the **lineage** of your project, you can use the dbt
+  [`ref()`](https://docs.getdbt.com/reference/dbt-jinja-functions/ref) function.
+
+- To keep your models **up-to-date** in Materialize, you can index the view. In
+  Materialize, [indexes](/concepts/indexes/) on a view maintain view results in
+  memory within a [cluster](/concepts/clusters/ "pools of compute resources
+  (CPU, memory, and scratch disk space)"). As the underlying data changes,
+  indexes **incrementally update** the view results in memory. To index a view,
+  see [Configuration: Indexes](#indexes).
+
+{{</ tip >}}
```

Comment:
Once https://github.com/MaterializeInc/materialize/pull/29090 is merged, will also link couple of times from this page.